src/fragments/lib/push-notifications/android/getting_started/10_pre_req.mdx

```diff
@@ -0,0 +1,5 @@
+* An Android application targeting at least Android SDK API level 24 with Amplify libraries integrated
```

Comment:
In here do we mean target SDK version or minimum SDK version because both can mean different things and this sentence is feels a bit confusing.